packages/dx-react-grid/docs/reference/table-group-header.md

```diff
@@ -0,0 +1,64 @@
+# TableGroupHeader Plugin Reference
```

Comment:
I'd prefer using the 'band' word in the name to avoid possible confusion between this and grouping features.